comp/metadata/packagesigning/README.md

```diff
@@ -0,0 +1,58 @@
+# Package Signing Payload
+
+This component populates the Linux Packages signing information in the `inventories` product in DataDog. They fill the `host_signing` table.
+
+This is enabled by default but can be turned off using `inventories_enabled` config.
+
+The payload is sent every 12h (see `inventories_max_interval` in the config). It has a different collection timeframe than the other inventory payloads.
+
+# Format
+
+The payload is a JSON dict with a list of keys, each having the following fields
+
+- `hostname` - **string**: the hostname of the agent as shown on the status page.
+- `timestamp` - **int**: the timestamp when the payload was created.
+- `signing_keys` - **list of dict of string to JSON type**
+  - `fingerprint` - **string**: the 16-char long key fingerprint.
```

Comment:
How strong of a validation for uniqueness do we need? [16-char fingerprint should not be assumed as unique](https://www.rfc-editor.org/rfc/rfc4880#section-3.3).

Reply:
I moved to 40-char fingerprint instead